packages/ods/src/components/link/documentation/migration.from.17.x.mdx

```diff
@@ -0,0 +1,30 @@
+## Attributes changes
+
+`contrasted ` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Icons are now font, they will by default have the same color as the current text. (TODO will be true after font update only)
+
+`slot ` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+You can use the new `label` attribute to obtain the same behavior.
+
+## Migration examples
```

Comment:
I think we should have a bit more explanation for the slot here.
Basically, we should have an explanation for any change that might confuse our user.